src/profile-logic/tracks.js

```diff
@@ -56,11 +58,20 @@ const GLOBAL_TRACK_DISPLAY_ORDER = {
 function _getDefaultLocalTrackOrder(tracks: LocalTrack[]) {
   const trackOrder = tracks.map((_, index) => index);
   // In place sort!
-  trackOrder.sort(
-    (a, b) =>
+  trackOrder.sort((a, b) => {
+    // Sort by the threadIndex first, putting things without a threadIndex at
+    // the top; then, sort by the local display order. This helps to ensure
+    // that IPC tracks are grouped below the thread they correspond to.
+    const threadA = 'threadIndex' in tracks[a] ? tracks[a].threadIndex : -1;
+    const threadB = 'threadIndex' in tracks[b] ? tracks[b].threadIndex : -1;
+    if (threadA !== threadB) {
```

Comment:
I'm a bit torn here on the implementation. The current metaphor for tracks is that each track is either a global track, or a local track. A global track is an individual process, or something like a screenshots. The local tracks are something that is going on in that process. Having the IPC tracks as a local track is somewhat confusing this breakdown.

Would it be possible to create 1 local track for the entire process's IPC? Then inside that track, each thread would have a separate breakdown.